espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/ComposeNodeElement.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.model
+
+import android.graphics.Rect
+import androidx.compose.ui.semantics.*
+import androidx.compose.ui.test.*
+import io.appium.espressoserver.EspressoServerRunnerTest
+import java.lang.AssertionError
+
+class ComposeNodeElement(private val node: SemanticsNode) {
+
+    val contentDescription: CharSequence?
+        get() = node.config.getOrNull(SemanticsProperties.ContentDescription)?.get(0)
```

Comment:
we should verify the array length here

Reply:
I was thinking about we could have a `firstObject` extension method  on a List class. This method should be safe to call (e.g.) return null if the list has no elements or the actual value of the first element and will allow us to reduce the amount of the boilerplate code

Reply:
I found there is already such a method https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/first-or-null.html